<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_trabajadores</class>
 <widget class="QWidget" name="main_trabajadores">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trabajadores</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::AddressBookNew">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="windowOpacity">
   <double>10.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
text-align: center; 
qproperty-alignment: AlignCenter;
}

#main_trabajadores{
background-color: rgb(8,8,8);
}</string>
  </property>
  <widget class="QFrame" name="frame_busquedad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>741</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 9, 36);
border-radius: 10;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <widget class="QLineEdit" name="input_busquedad">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#input_busquedad {
                background-color: #fff;
                color: #000;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Trabajadores</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_lista_trabajadores">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>741</width>
     <height>351</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 9, 36);
border-radius: 10;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lista de trabajadores</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_lista">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>711</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 15, 61);
border-radius: 15;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <widget class="QFrame" name="frame_trabajadores">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>691</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background-color: rgb(28, 21, 84);
border-radius: 18;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <widget class="QLabel" name="label_nombre_trabajadores">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;</string>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_email_trabajadores">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_eliminar">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#push_eliminar{
background-color: rgb(138, 0, 0);
border-radius: 20px;
color: #fff;
}

#push_eliminar:hover{
background-color: rgb(255, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_editar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#push_editar{
background-color: rgb(0, 170, 0);
border-radius: 20px;
color: #fff;
}

#push_editar:hover{
background-color: rgb(0, 247, 0);
}</string>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 9, 36);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 255);
border-radius: 20;
padding: 3px;
font-size: 5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources/resources.qrc">
      <normaloff>:/icons/icons/discord.svg</normaloff>:/icons/icons/discord.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="push_maxiniza">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_maxiniza {
border: none;
	border-radius: 15px;
}

#push_maxiniza:hover {
	background-color: rgb(124, 124, 124);
}

#push_maxiniza:pressed {
	background-color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources/resources.qrc">
      <normaloff>:/icons/icons/maximize.svg</normaloff>:/icons/icons/maximize.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="push_close">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_close {
	border: none;
	border-radius: 15px;
}

#push_close:hover {
	background-color: rgb(124, 124, 124);
}

#push_close:pressed {
	background-color: red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources/resources.qrc">
      <normaloff>:/icons/icons/close.svg</normaloff>:/icons/icons/close.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="push_miniza">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_miniza{
	border-radius: 15px;
border: none;
}

#push_miniza:hover {
	background-color: rgb(124, 124, 124);
}

#push_miniza:pressed {
	background-color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources/resources.qrc">
      <normaloff>:/icons/icons/minus.svg</normaloff>:/icons/icons/minus.svg</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_titulo">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>79</y>
     <width>741</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 9, 36);
border-radius: 10;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
font-size: 25px;</string>
    </property>
    <property name="text">
     <string>Menu de trabajadores</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
